src/test/features/response/draft/responseDraft.ts

```diff
@@ -32,6 +33,7 @@ describe('ResponseDraft', () => {
       expect(responseDraftModel.moreTimeNeeded.option).to.eql(MoreTimeNeededOption.YES)
       expect(responseDraftModel.isMoreTimeRequested()).to.be.eql(true)
       expect(responseDraftModel.impactOfDispute.text).to.equal('This dispute has affected me badly, I cried')
+      expect(responseDraftModel.statementOfMeans.residence.type.value).to.equal(ResidenceType.OTHER.value)
```

Comment:
Would `expect(responseDraftModel.statementOfMeans.residence.type).to.equal(ResidenceType.OTHER)` work?